packages/carbon-component-mapper/src/tests/text-field.test.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+
+import { FormRenderer, componentTypes } from '@data-driven-forms/react-form-renderer';
+
+import FormTemplate from '../form-template';
+import componentMapper from '../component-mapper';
+import { TextInput, NumberInput } from 'carbon-components-react';
+
+describe('<TextInput and NumberInput />', () => {
+  let onChange;
+  let wrapper;
+  let submitSpy;
+  beforeEach(() => {
+    onChange = jest.fn();
+    submitSpy = jest.fn();
+  });
+  afterEach(() => {
+    onChange.mockReset();
+    submitSpy.mockReset();
+  });
+  it('renders NumberInput', () => {
+    const schema = {
+      fields: [
+        {
+          component: componentTypes.TEXT_FIELD,
+          name: 'input',
+          label: 'Please enter a value',
+          type: 'number',
+          step: 1,
+          initialValue: 1,
+          onChange
```

Comment:
Oh, this is the issue. You are still overwriting the onychange event.

Reply:
ah ok. i will try to remove that and see, thanks.